src/lib.rs

```diff
@@ -3415,11 +3415,11 @@ impl Build {
             })
             .map(|prefix| *prefix)
             .or_else(||
-            // If no toolchain was found, provide the first toolchain that was passed in.
-            // This toolchain has been shown not to exist, however it will appear in the
-            // error that is shown to the user which should make it easier to search for
-            // where it should be obtained.
-            prefixes.first().map(|prefix| *prefix))
+                // If no toolchain was found, provide the first toolchain that was passed in.
```

Comment:
I'd recommend moving the comment outside the closure

Reply:
Thanks!

Modified and execute cargo fmt.

Please review again.